### ISSUE TITLE
2.17.0 broken import

### ISSUE THREAD

**Issue by tysonholub** [chargebee/chargebee-python#53]
release of 2.17.0 appears to have broken imports

```
$ poetry run pip install chargebee==2.17.0 -U
Collecting chargebee==2.17.0
  Downloading chargebee-2.17.0.tar.gz (141 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 141.9/141.9 kB 2.6 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Requirement already satisfied: requests in ./.venv/lib/python3.10/site-packages (from chargebee==2.17.0) (2.28.1)
Requirement already satisfied: charset-normalizer<3,>=2 in ./.venv/lib/python3.10/site-packages (from requests->chargebee==2.17.0) (2.1.0)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in ./.venv/lib/python3.10/site-packages (from requests->chargebee==2.17.0) (1.26.11)
Requirement already satisfied: certifi>=2017.4.17 in ./.venv/lib/python3.10/site-packages (from requests->chargebee==2.17.0) (2022.6.15)
Requirement already satisfied: idna<4,>=2.5 in ./.venv/lib/python3.10/site-packages (from requests->chargebee==2.17.0) (3.3)
Building wheels for collected packages: chargebee
p  Building wheel for chargebee (setup.py) ... done
  Created wheel for chargebee: filename=chargebee-2.17.0-py3-none-any.whl size=168044 sha256=d9f58a2f164b1a2497a3e1caaf9c822c88ba923be664285d41d58da626c77852
  Stored in directory: /home/tholub/.cache/pip/wheels/ff/1c/70/0ae76d8ba65b08c2351acdb3f0942da8fd391155f4e9b11d16
Successfully built chargebee
Installing collected packages: chargebee
  Attempting uninstall: chargebee
    Found existing installation: chargebee 2.16.0
    Uninstalling chargebee-2.16.0:
      Successfully uninstalled chargebee-2.16.0
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
flask-subscription 0.1.0 requires chargebee==2.16.0, but you have chargebee 2.17.0 which is incompatible.
Successfully installed chargebee-2.17.0

[notice] A new release of pip available: 22.1.2 -> 22.2.1
[notice] To update, run: pip install --upgrade pip
tholub@pop-os:~/git/mobius-2/flask_subscription migrate.registry (fb9412b)$ poetry run python
Python 3.10.4 (main, May 19 2022, 15:17:01) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from chargebee import Content
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Content' from 'chargebee' (/home/tholub/git/mobius-2/flask_subscription/.venv/lib/python3.10/site-packages/chargebee/__init__.py)
```
This failed import is issued [here](https://github.com/chargebee/chargebee-python/blob/v2.17.0/chargebee/models/hosted_page.py#L13)

**Comment by cb-yateshmathuria** [comment]
@tysonholub - Thank you for notifying. Fixed in [v2.17.1](https://github.com/chargebee/chargebee-python/tree/v2.17.1)

**Comment by cb-yateshmathuria** [comment]
Closing this issue. Let us know if it worked for you.
